src/Explorer/Panes/PublishNotebookPaneAdapter.tsx

```diff
@@ -158,10 +178,22 @@ export class PublishNotebookPaneAdapter implements ReactAdapter {
       clearFormError: this.clearFormError
     };
 
-    return <PublishNotebookPaneComponent {...publishNotebookPaneProps} />;
+    return !this.acceptedCodeOfConduct ? (
+      <div style={{ padding: "15px", marginTop: "10px" }}>
+        <CodeOfConductComponent
+          junoClient={this.junoClient}
+          onAcceptCodeOfConduct={() => {
+            this.acceptedCodeOfConduct = true;
+            this.triggerRender();
+          }}
+        />
+      </div>
+    ) : (
+      <PublishNotebookPaneComponent {...publishNotebookPaneProps} />
+    );
   };
 
-  private reset = (): void => {
+  private reset = async (): Promise<void> => {

```

Comment:
I'm not seeing any async call inside. Or am I missing something? If it's to confirm to some signature in the caller, add a comment.

Reply:
reverted it